src/Pages/OATEditorPage/OATEditorPage.types.ts

```diff
@@ -29,5 +34,5 @@ export interface IOATEditorState {
     edge?: string;
     confirmDeleteType?: string;
     confirmDeletePayload?: any;
```

Comment:
Do we still need the two above properties?

Reply:
Already deleted everything related to those